scripts/coverage/config.json

```
@@ -11,6 +11,7 @@
     ".*Preview[.]swift",
     "\\/Views\\/",
     "Core\/Common\/CommonUI\/EmbeddedWebPage\/ViewModel\/EmbeddedWebPageViewModel.swift",
+    "Core\/Core\/Common\/Extensions\/InstColorExtensions.swift",
```

Please keep the alphabetical ordering and
```suggestion
    "Core\/Common\/Extensions\/InstColorExtensions.swift",
```